src/main/kotlin/calculator/Operator.kt

```diff
@@ -0,0 +1,20 @@
+package calculator
+
+enum class Operator(val prime: String) {
+    ADD("+"),
+    SUBTRACT("-"),
+    MULTIPLY("*"),
+    DIVIDE("/");
+
+    companion object {
+        fun findByValue(prime: String): Operator {
+            Operator.values()
+                .forEach { operator ->
+                    if (prime == operator.prime) {
+                        return operator
+                    }
+                }
```

Comment:
```suggestion
            values().find { operator ->
                prime == operator.prime
            }
```
Operator에 정의된 연산자인지 아닌지 찾는 함수를 잘 구현해주셨네요. 👍 
정의된 연산자를 찾기 위해 forEach를 활용해주셨는데요.
forEach를 활용해주셔도 좋지만, find 함수를 사용하면 좀 더 간결하게 코드를 작성하실 수 있어요. 😄